lib/interface/cli/helpers/general.js

```diff
@@ -106,6 +106,27 @@ const prepareKeyValueObjectsFromCLIEnvOption = (environmentVariables) => {
     return variables;
 };
 
+const prepareKeyValueObjectsFromEnvFileOption = (environmentVariables) => _.map(environmentVariables, (value, key) => {
```

Comment:
let's add documentation for these new functions similarly to what was done in `prepareKeyValueObjectsFromCLIEnvOption` func so it will be easier to understand the differences between them